src/test/java/winstone/HostConfigurationTest.java

```diff
@@ -0,0 +1,41 @@
+package winstone;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.NavigableMap;
+import java.util.Properties;
+import java.util.TreeMap;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ErrorCollector;
+
+public class HostConfigurationTest {
+
+    @Rule
+    public ErrorCollector errors = new ErrorCollector();
+
+    @Test
+    public void mimeTypes() throws IOException {
```

Comment:
This test ensures that we are only ever adding to the Jetty default MIME types, never replacing them. When upstream Jetty has a MIME type, we want to use it whenever possible. This is not only so that we have less code to maintain, but also so that we avoid bugs like JENKINS-73381.